v0100/srclib/dpstub.asm

```diff
@@ -884,3 +892,7 @@ regs_len equ $ - regs
 fbuf        resb FBUFSZ ; shared with 32-bit code
 
 alignb 16
+STKSZ equ 1024
```

Comment:
Move this equ line to immediately after the line with `FBUFSZ equ` at the beginning of the file.

Also, change the value to 4096. 1024 may be too small.

Reply:
Done, although I don't really think
4096 is necessary. As I understand,
this stack is only used by the stub
itself, and there are no places in that
stub that consumes that much.
Or what am I missing?
Anyway, I changed to 4096, as you requested.

Reply:
True, however, there are interrupt handlers using that same stack. I'd rather not run into a situation when a device driver corrupts memory when it runs out of stack space.

Reply:
OK.

Reply:
> True, however, there are interrupt handlers using that same stack.

Usually not, unless you set
`STACKS=0` in config.sys.
Anyway, all done, so can this
be applied?